vm/src/hint_processor/builtin_hint_processor/bootloader_hints.rs

```diff
@@ -106,6 +110,81 @@ pub fn prepare_simple_bootloader_output_segment(
     Ok(())
 }
 
+/*
+Implements hint:
+%{
+    output_start = ids.output_ptr
+%}
+*/
+pub fn save_output_pointer_hint(
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    let output_ptr = get_ptr_from_var_name("output_ptr", vm, ids_data, ap_tracking)?;
+    exec_scopes.insert_value("output_start", output_ptr);
+    Ok(())
+}
+
+/*
+Implements hint:
+%{
+    packed_outputs = bootloader_input.packed_outputs
+%}
+*/
+pub fn save_packed_outputs_hint(
+    _vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    _ids_data: &HashMap<String, HintReference>,
+    _ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    let bootloader_input = exec_scopes.get("bootloader_input")?;
+    let packed_outputs = bootloader_input; // TODO: need type for bootloader_input / query its packed_outputs field
+    exec_scopes.insert_value("packed_outputs", packed_outputs);
+    Ok(())
+}
+
+/*
+Implements hint:
+%{
+    packed_outputs = packed_output.subtasks
+%}
+*/
+pub fn set_packed_output_to_subtasks_hint(
+    _vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    _ids_data: &HashMap<String, HintReference>,
+    _ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    let packed_outputs = exec_scopes.get("packed_outputs")?;
+    let subtasks = packed_outputs; // TODO: need type for packed_output / query its subtasks field
+    exec_scopes.insert_value("packed_outputs", subtasks);
+    Ok(())
+}
+
+/*
+Implements hint:
+%{
+    data = packed_output.elements_for_hash()
+    ids.nested_subtasks_output_len = len(data)
+    ids.nested_subtasks_output = segments.gen_arg(data)";
+%}
+*/
+pub fn guess_pre_image_of_subtasks_output_hash_hint(
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    let packed_outputs = exec_scopes.get::<Relocatable>("packed_outputs")?;
```

Comment:
Implement `PackedOutput` as an empty struct for now. We'll modify it when we know what it is, but it's better to have the correct types already.